### PR TITLE
feat: Add horizontal variations

### PR DESCRIPTION
**Pull request by anhanh11001**
Details:
- Add horizontal variations for saved configurations view, main activity view
- Extract resources
Fixes: #209

Screenshots for the change:

### CODE REVIEW

File: app/src/main/res/layout-land/recycler_save_item.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            android:layout_height="wrap_content"
+            android:padding="@dimen/spacing_small">
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/spacing_normal"
+                android:gravity="center"
+                android:minHeight="20dp"
```
Comment (yashk2000) [inline app/src/main/res/layout-land/recycler_save_item.xml]: Please replace this with dimens resources. Avoid hardcoding dimensions.

File: app/src/main/res/layout-land/recycler_save_item.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                android:text="AKD"
+                android:textAppearance="@style/TextAppearanceList"
+                android:textSize="20sp" />
```
Comment (yashk2000) [inline app/src/main/res/layout-land/recycler_save_item.xml]: Here too.

File: app/src/main/res/layout-land/recycler_save_item.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:id="@+id/options"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:minHeight="40dp"
```
Comment (yashk2000) [inline app/src/main/res/layout-land/recycler_save_item.xml]: Here also.

Comment (anhanh11001) [reply app/src/main/res/layout-land/recycler_save_item.xml]: done!

**Pull request merged**